packages/data-stores/src/site/types.ts

```diff
@@ -128,6 +128,7 @@ export interface SiteDetails {
 	visible?: boolean;
 	wpcom_url?: string;
 	user_interactions?: string[];
+	is_at?: boolean;
```

Comment:
Not sure what is this and where it is used 🤔

Reply:
Good catch, it isn't anymore.